### ISSUE TITLE
Shouldn't `league/commonmark` be a non-dev requirement?

### ISSUE THREAD

**Issue by LordSimal** [tempestphp/highlight#72]
Currently it is a dev-requirement which means, it doesn't get installed if doing a `composer install -o --no-dev` for a production build.
https://github.com/tempestphp/highlight/blob/371413ae6056626b276dee508afdaa5f28711827/composer.json#L27

**Comment by brendt** [comment]
No, it's included for testing, but you don't need to use commonmark if you don't want to. If you need commonmark support, you'll have to require it in the project: https://github.com/tempestphp/highlight?tab=readme-ov-file#commonmark-integration
